ROADMAP.md

```diff
@@ -1,6 +1,20 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.5 Release, Due: March 2022
+## Kubeflow 1.6 Release, Due: July 2022
+* Kubeflow 1.6 [milestones and timelines](https://github.com/kubeflow/community/pull/558)
+
```

Comment:
Adding a list of a few features shared by the WG during the April 26th Community meeting for WG pipelines, katib, training operators, and kserve.

```suggestion
### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)

#### Kubeflow Pipelines, v x.y.z
* V2 Preview: Support running pipeline in Argo-agnostic approach
* New DAG visualization based on Pipeline Template and MLMD
* SDK to change from `kfp.v2` to `kfp`
* KFP SDK CLI design

#### Katib, v x.y.z
* Support for Population based training
* Support JSON format for Metrics Collector
* Support for Kubernetes 1.22
* UI Enhancement to add a separate page for Trials and show Trial logs

#### Training Operator, v x.y.z
* PyTorch elastic training enhancements
* Elastic scheduling with Volano support
* Support for Kubernetes 1.22
* MPI Operator v2

#### Notebooks, Central Dashboard, Web Apps, Manifest, v x.y.z
* TBA

#### KServe, v x.y.z
* Knative 1.0 support and certified for KServe Serverless installation
* KServe 0.9 SDK with ModelServer API name changes and improvement for CloudEvent support
* New ServingRuntime and ClusterServingRuntime CRDs
* New ModelSpec in InferenceServices
* ModelMesh updates include support for multi-namespace reconciliation

```

Reply:
@annajung line 21 - Volcano is missing a "c"

Reply:
fixed! thanks

Reply:
the notebook WG features have been added to the ROADMAP. Feel free to accept the suggestion to show the content in the PR for an easier way for WG to leave feedback.